cli/internal/cmd/init/init.go

```diff
@@ -139,6 +139,12 @@ func replaceText(templateContext TemplateContext) error {
 		return err
 	}
 
+	// remove .git folder
+	err = os.RemoveAll(path + "/.git")
```

Comment:
this form of concatenation is not preffered due to os. issues. Use path builder instead

Reply:
sure!! thanks.